lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb

```diff
@@ -60,6 +63,8 @@ def custom_build_settings
               'PODS_ROOT'                          => '$(SRCROOT)',
               'PRODUCT_BUNDLE_IDENTIFIER'          => 'org.cocoapods.${PRODUCT_NAME:rfc1034identifier}',
               'SKIP_INSTALL'                       => 'YES',
+
+              'ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES' => 'NO',
```

Comment:
nit: either remove the space above or add a comment explaining why this is needed. I assume static libs cannot "embed" anything, but also dynamic frameworks don't need to embed the standard libs since the user/app target will?